nekoyume/Assets/_Scripts/UI/Widget/System/VersionSystem.cs

```diff
@@ -39,14 +42,13 @@ private void SubscribeBlockHash(BlockHash hash)
 
         private void UpdateText()
         {
-            const string format = "APV: {0} / #{1} / Hash: {2}";
             var hash = _hash.ToString();
-            var text = string.Format(
-                format,
-                _version,
-                _blockIndex,
-                hash.Length >= 4 ? hash.Substring(0, 4) : "...");
-            informationText.text = text;
+            hash = hash.Length >= 4 ? hash.Substring(0, 4) : "...";
+#if UNITY_ANDROID || UNITY_IOS
+            informationText.text = $"APV: {_version} / #{_blockIndex} / Hash: {hash} / ver: {UnityEngine.Application.version}";
```

Comment:
There is client hash data and you can get like below.
```cs
var clientCommitHash = Resources.Load<TextAsset>("ClientHash")?.text[..8] ?? string.Empty;
```
Could you add this info beside to `ver`?

Reply:
Hmm... I don't think this is required yet.🤔